testharness2.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:alpine
+
+RUN apk update && \
+    apk upgrade && \
+    apk add --no-cache bash git openssh
+
+ENV PORT=80
+EXPOSE 80
+RUN npm install serve@11.3.0 -g
+WORKDIR /var/web/
+ENTRYPOINT ["npx", "--no-install", "serve", "-c", "serve-test.json", "-p", "80", "/var/web/"]
+
+ADD serve-test.json /var/web/
```

Comment:
why `/var/`?

Reply:
It's a common practices to use `/var/` to store "variables", i.e. things that can change while the system is running.

It's not great to put everything at root, e.g. `/web/`.